src/components/pages/contact/form.tsx

```diff
@@ -14,18 +15,46 @@ const StyledLink = styled(Link)`
   }
 `;
 
-export const Form = () => (
-  <>
-    <SectionHeader headline={'E-Mail'}>
-      <Trans i18nKey="contact.info-link">
-        <p>
-          Nutzen Sie unser Kontaktformular oder schreiben Sie uns eine E-Mail an
-          <StyledLink to="mailto:info@satellytes.com">
-            info@satellytes.com
-          </StyledLink>
-        </p>
-      </Trans>
-    </SectionHeader>
-    <ContactForm />
-  </>
-);
+const StyledSectionHeader = styled(SectionHeader)`
+  margin-bottom: 48px;
+`;
+
+export const Form = () => {
```

Comment:
This is kind of upside down.
You export this component `Form` and internally you refer to a `ContactForm`. I would have expected to have an internal `Form` and you export `ContactForm`. Maybe you can swap the names ?